{
	"id": "CVE-2023-40687",
	"sourceIdentifier": "psirt@us.ibm.com",
	"vulnStatus": "Awaiting Analysis",
	"published": "2023-12-04T02:15:07.077",
	"lastModified": "2023-12-04T02:58:50.030",
	"descriptions": [
		{
			"lang": "en",
			"value": "IBM DB2 for Linux, UNIX and Windows (includes Db2 Connect Server) 10.5, 11.1, and 11.5 is vulnerable to denial of service with a specially crafted RUNSTATS command on an 8TB table.  IBM X-Force ID:  264809."
		}
	],
	"references": [
		{
			"source": "psirt@us.ibm.com",
			"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/264809"
		},
		{
			"source": "psirt@us.ibm.com",
			"url": "https://www.ibm.com/support/pages/node/7087149"
		}
	],
	"metrics": {
		"cvssMetricV31": [
			{
				"source": "psirt@us.ibm.com",
				"type": "Secondary",
				"cvssData": {
					"version": "3.1",
					"vectorString": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:N/I:N/A:H",
					"attackVector": "NETWORK",
					"attackComplexity": "HIGH",
					"privilegesRequired": "LOW",
					"userInteraction": "NONE",
					"scope": "UNCHANGED",
					"confidentialityImpact": "NONE",
					"integrityImpact": "NONE",
					"availabilityImpact": "HIGH",
					"baseScore": 5.3,
					"baseSeverity": "MEDIUM"
				},
				"exploitabilityScore": 1.6,
				"impactScore": 3.6
			}
		]
	},
	"weaknesses": [
		{
			"source": "psirt@us.ibm.com",
			"type": "Primary",
			"description": [
				{
					"lang": "en",
					"value": "CWE-20"
				}
			]
		}
	]
}
